testsuite/features/support/constants.rb

```diff
@@ -710,15 +710,15 @@
         el9-manager-tools-pool-x86_64-rocky
         el9-manager-tools-updates-x86_64-rocky
       ],
-    'rockylinux-8' =>
+    'rockylinux8' =>
       %w[
-        rockylinux-8-x86_64
-        rockylinux-8-appstream-x86_64
+        rockylinux8-x86_64
+        rockylinux8-appstream-x86_64
       ],
-    'rockylinux-9' => # CHECKED
+    'rockylinux9' => # CHECKED
```

Comment:
I marked it as "Checked" so it should have been correct... However, it's possible i missed an issue with the dash (????)

Reply:
I don't know, please refer to my comment above https://github.com/uyuni-project/uyuni/pull/8367#discussion_r1504358741 or check it by yourself. Something might have changed since you have added it.

Reply:
see above

Reply:
Result seems to be positive, very similar to https://github.com/uyuni-project/uyuni/pull/8367#discussion_r1507655478
Full file for reference: 
[rockylinux9-x86_64.log](https://github.com/uyuni-project/uyuni/files/14449144/rockylinux9-x86_64.log)

Reply:
then fine. yeah could have change, or we did a typo, or whatever, we might never know.